packages/design-picker/src/components/generated-design-picker.tsx

```diff
@@ -77,24 +78,69 @@ const GeneratedDesignPicker: React.FC< GeneratedDesignPickerProps > = ( {
 } ) => {
 	const { __ } = useI18n();
 
+	/* eslint-disable @typescript-eslint/no-unused-vars */
+	const wrapperRef = useRef< HTMLDivElement >( null );
+	const viewmoreRef = useRef< HTMLDivElement >( null );
+	const propertyForThumbnailHeight = '--thumbnail-design-height';
+
+	useEffect( () => {
+		const onResize = () => {
```

Comment:
I think we can just calculate the height of the `generated-design-picker__thumbnails`. That being said, its height should be `100vh - the offsetTop of the element`. Then, we can use CSS to set the min-height and max-height of the `GeneratedDesignThumbnail` to make it responsive.

Besides, the responsive thumbnails are only applied on the desktop. So, you still need to handle this part.

Reply:
I have to calculate the actual height of the thumbnails manually since it is required to keep the viewmore button to stay visible on the viewport, so the max height of `generated-design-picker__thumbnails` is based on
- Viewport height
- The element offsetTop, I added scrollY just to be safe regarding to scrolling `thumnbnailWrapper.getClientRects()[ 0 ].y + window.scrollY`
- The `generated-design-picker__view-more-wrapper`, to make sure it will stay within viewport

Reply:
@arthur791004 I made sure we only update the thumbnails container's height on resize, the thumbnails button height is generated using css

Reply:
Okay, I'll move it to ready for review!